Solutions/Tomcat/Hunting Queries/TomcatERRORs.yaml

```diff
@@ -0,0 +1,23 @@
+id: 7be944be-487f-11ec-81d3-0242ac130003
+name: Tomcat - Catalina errors
+description: |
+  'Query shows errors events.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1070
+query: |
+  TomcatEvent
+  | where TimeGenerated > ago(24d)
+  | where EventMessage contains "error"

```

Comment:
Use HAS if there are any types of characters that are not alpha numeric, such as spaces, slashes, dashes and so on.

Reply:
fixed